### PR TITLE
Setting up Hello-Microverse project with Html&Css

### PR DESCRIPTION
**Pull request by tsigewossne**
### Basic Files
- A .html file that includes a header with the text "Hello Microverse!". which uses the h1 tag for that.
- A . CSS file that includes one class with styles for the header (Aqua color).
- A test. md file with the text "This file should be ignored by git".
### .gitignore file to include files that should be ignored from the repository

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+
+<!-- PROJECT DESCRIPTION -->
+
```
Comment (Shedrack-Sunday) [inline README.md]: - [ ] The header section here is complete. you're missing a key section which is the LICENSE file. Kindly add it up here, at the root of your project, and also the path at the bottom as it's done on the template.
- The file is very important to show ownership

## Header:
image

## Path:
image

File: test.md [hunk 2]
```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```
Comment (Shedrack-Sunday) [inline test.md]: - [ ] This file should be ideally ignored by git. As such, it should not be committed to your project. Kindly re-add the test file in `.gitignore`, and make sure this does not show, as that's the essence of the `.gitignore` file

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @tsigewossne       👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing some of the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 


## Highlights 
- Nice setup of linters✔️  
- Good job with using GitHub features  ✔️  
  
# Required Changes  🌵 

- [ ] There are a few things you need to take note of, you have a great project as expected 👍 

_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by tsigewossne** [comment]
@Shedrack-Sunday Thank you for your supportive comments and suggestions. I have made the following changes 

- Add a key section of the License on the README.md file
- Remove the test.md file manually and add it to .gitignore Again so that it is ignored by the repository

**Review by Gambit142 — APPROVED** [review]
Hi @tsigewossne 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- All linters checks passed ✔️
- Commit message is capitalized and in imperative tense ✔️ 
- It has HTML and CSS files ✔️ 
- It has **_node_module_** and _**test.md**_ git ignored ✔️ 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**